gnss_ins_sim/pathgen/pathgen.py

```diff
@@ -176,7 +176,7 @@ def path_gen(ini_pos_vel_att, motion_def, output_def, mobility, ref_frame=0, mag
     idx_low_freq = 0        # data index for gps, odo
     for i in range(0, motion_def.shape[0]):
         com_type = round(motion_def[i, 0])      # command type of this segment
-        gps_visibility = motion_def[i, 8]       # gps visibility       
```

Comment:
This is just due to my editor, and can be ignored.

Reply:
Hi theophilec,

This is warning is from pylint, I think. I tried to follow pylint. But I missed this place. Thank you for poiting this out.

Dong